src/test/java/guide/UsingHttpsTest.java

```diff
@@ -56,29 +68,133 @@ public void shouldPassWhenExpectedStubDidHappen() throws GeneralSecurityExceptio
 
     @Test
     public void shouldBePossibleToSpecifyKeyStoresWithStandardProperties() throws GeneralSecurityException, IOException {
+        // use a different key store
+        System.setProperty("javax.net.ssl.keyStore", "build/resources/test/keystore_server_test");
```

Comment:
Hardcoding `build` folder here and in other places doesn't feel right

Reply:
I thought about it. I deliberately did not use class resource as I treat this as if it is outside restito project. In case one wants to use a different certificate, the key/certificate may not necessary stored as class resources.

Reply:
Ok. A matter of preference, I guess. But I think your point becomes clearer even without hardcoding if we don't mix build-in and test-only key stores in the test code (like suggested in https://github.com/mkotsur/restito/pull/84/commits/f5df326451c5d1985a65a205e7e837ece8d8fb9a#r979228905)

Reply:
Yeah, just a matter of preference. I am good with your refactored version.